score-xml/src/main/java/io/cloudslang/content/xml/entities/inputs/CustomInputs.java

```diff
@@ -84,7 +85,7 @@ public CustomInputsBuilder withQueryType(String inputValue) {
         }
 
         public CustomInputsBuilder withDelimiter(String inputValue) {
-            if(inputValue == null || inputValue.isEmpty()){
+            if(StringUtils.isEmpty(inputValue)){
```

Comment:
@SamMarkowitz  here I suggest to use StringUtils.isBlank(inputValue) because will check to see if inputValue is null, empty or a string that contains nothing but blank chars.